code/datums/traits/good.dm

```diff
@@ -426,6 +485,41 @@
 	lose_text = "<span class='danger'You feel like you're allowed to fly on planes again.</span>"
 	locked = TRUE
 
+/datum/quirk/advanced_explosive_crafting/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	// I made the quirks add the same recipes as the trait books. Feel free to nerf this
+	H.mind.learned_recipes += list(
+		/datum/crafting_recipe/incendiary, 
```

Comment:
This can be broken up into two global lists, one for the basic explosive crafting recipes, and one for what the advanced explosive crafting adds to that, and have the two lists added just like this.